sdk/automanage/Microsoft.Azure.Management.AutoManage/src/AzSdk.RP.props

```diff
@@ -0,0 +1,7 @@
+﻿<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <!--This file and it's contents are updated at build time moving or editing might result in build failure. Take due deligence while editing this file-->
+  <PropertyGroup>
+    <AzureApiTag>Automation_2015-10-31;Automation_2018-01-15;Automation_2017-05-15-preview;Automation_2018-06-30;</AzureApiTag>
```

Comment:
This should be your automanage api-version.

Reply:
Good catch, fixed!